lib/fastifySession.js

```diff
@@ -62,13 +62,14 @@ function decryptSession (sessionId, options, request, done) {
         newSession(secret, request, cookieOpts, idGenerator, done)
         return
       }
-      if (session?.expires && session.expires <= Date.now()) {
+      if (session?.cookie?.expires && session.cookie.expires <= new Date()) {
```

Comment:
```suggestion
      if (session?.cookie?.expires && session.cookie.expires <= new Date().getTime()) {
```

I prefer being explicit 😀

Reply:
```suggestion
      if (session?.cookie?.expires?.getTime && session.cookie.expires.getTime() <= new Date().getTime()) {
```

Reply:
I don't think the `getTime` of `session?.cookie?.expires?.getTime` is needed - if `expires` is defined so is its `getTime`. But beyond that I agree 👍

Reply:
you should prefer `Date.now()` over `new Date().getTime() `

But on the other hand,  you can compares Dates directly. So if session.cookie.expires is a Date you can compare it with Date.now()

Reply:
```suggestion
      if (session.cookie?.expires && session.cookie.expires <= Date.now()) {
```